src/DI/OrmXmlExtension.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types = 1);
+
+namespace Nettrine\ORM\DI;
+
+use Doctrine\ORM\Configuration;
+use Doctrine\ORM\Mapping\Driver\XmlDriver;
+use Nette\DI\CompilerExtension;
+use Nette\DI\Helpers;
+use Nettrine\ORM\Exception\Logical\InvalidStateException;
+
+class OrmXmlExtension extends CompilerExtension
+{
+
+	/** @var mixed[] */
+	public $defaults = [
+		'paths' => [], //'%appDir%'
```

Comment:
By default is `paths` field empty. So it won't work? What about suggest in readme to put there `%appDir%` or if `paths` is empty, fill `%appDir%` automatically?

Reply:
Sorry @f3l1x for such a late response, I have completely missed it.

To be honest i just copied this one from https://github.com/nettrine/orm/blob/master/src/DI/OrmAnnotationsExtension.php#L45 and I was not sure why it is in comment 😄